sled-agent/src/rack_setup/service.rs

```diff
@@ -1108,11 +1135,13 @@ impl ServiceInner {
                 nexus_address,
             )
             .await?;
+            rss_step.update(RssStep::Completed);
             return Ok(());
         } else {
             info!(self.log, "RSS configuration has not been fully applied yet");
         }
 
+        rss_step.update(RssStep::WaitingForPeers);
```

Comment:
Are we waiting for peers here? I think I'd be inclined to name this `CreateSledPlan`, although if multiple things happen inside `SledPlan::create()` maybe we should pass `rss_step` into it so it can note the things it does?